core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-create/dot-experiments-create.component.ts

```diff
@@ -0,0 +1,101 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+    FormBuilder,
+    FormControl,
+    FormGroup,
+    ReactiveFormsModule,
+    Validators
+} from '@angular/forms';
+import { InputTextareaModule } from 'primeng/inputtextarea';
+import { InputTextModule } from 'primeng/inputtext';
+import { SidebarModule } from 'primeng/sidebar';
+import { DotSidebarDirective } from '@portlets/shared/directives/dot-sidebar.directive';
+import { DotSidebarHeaderComponent } from '@shared/dot-sidebar-header/dot-sidebar-header.component';
+import { UiDotIconButtonModule } from '@components/_common/dot-icon-button/dot-icon-button.module';
+import { ButtonModule } from 'primeng/button';
+import { DotMessagePipeModule } from '@pipes/dot-message/dot-message-pipe.module';
+import { ActivatedRoute } from '@angular/router';
+import { DotFieldValidationMessageModule } from '@components/_common/dot-field-validation-message/dot-file-validation-message.module';
+import { Observable } from 'rxjs';
+import {
+    DotExperimentCreateStore,
+    DotExperimentsCreateStore
+} from '@portlets/dot-experiments/dot-experiments-create/store/dot-experiments-create-store.service';
+import { DotExperiment } from '@portlets/dot-experiments/shared/models/dot-experiments.model';
+
+interface CreateForm {
+    pageId: FormControl<string>;
+    name: FormControl<string>;
+    description: FormControl<string>;
+}
+
+@Component({
+    selector: 'dot-experiments-create',
+    standalone: true,
+    imports: [
+        CommonModule,
+        ReactiveFormsModule,
+
+        DotSidebarDirective,
+        DotSidebarHeaderComponent,
+        DotMessagePipeModule,
+        DotFieldValidationMessageModule,
+        UiDotIconButtonModule,
+
+        // PrimeNg
+        InputTextareaModule,
+        InputTextModule,
+        SidebarModule,
+        ButtonModule
+    ],
```

Comment:
+1

Reply:
Pass all 10 modules to a `DotExperimentsListModule` of functionality that maybe not will use (Create) instead of encapsulating it in a lazyloaded component.